drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -168,22 +168,21 @@ struct usbd_event {
 };
 
 /**
- * @brief Fifo element pool
+ * @brief Fifo element slab
  *	Used for allocating fifo elements to pass from ISR to work handler
  * TODO: The number of FIFO elements is an arbitrary number now but it should
  * be derived from the theoretical number of backlog events possible depending
  * on the number of endpoints configured.
  */
-#define FIFO_ELEM_MIN_SZ        sizeof(struct usbd_event)
-#define FIFO_ELEM_MAX_SZ        sizeof(struct usbd_event)
+#define FIFO_ELEM_SZ            sizeof(struct usbd_event)
 #define FIFO_ELEM_ALIGN         sizeof(unsigned int)
 
 #if CONFIG_USB_NRFX_EVT_QUEUE_SIZE < 4
 #error Invalid USBD event queue size (CONFIG_USB_NRFX_EVT_QUEUE_SIZE).
 #endif
```

Comment:
I know its not part of this PR but please remove this if You can. Its not needed, Kconfig will throw an error when invalid value is assigned. This option has `range 4 64`.

Reply:
done